serving-api/src/main/resources/reference.conf

```diff
@@ -36,8 +36,8 @@ sparkta {
 
   hdfs {
     hadoopUserName = stratio
-    hadoopConfDir = /home/ubuntu
     hdfsMaster = localhost
+    hdfsPort = 8020
```

Comment:
Please make this changes also in `dist/src/main/resources/application.conf` file